lib/rspec/its.rb

```diff
@@ -202,7 +198,7 @@ def should_not(matcher=nil, message=nil)
 
   # Add RSpec configuration setting to allow
   # the user to handle private method invoking warning
-  rspec.add_setting :its_private_method_debug
+  rspec.add_setting :its_raise_errors_for_private_method_calling
```

Comment:
Keeping in mind there's `configuration.raise_errors_for_deprecations!` I'm on the fence if we need this setting, or a deprecation message will be sufficient.